lib/rules/no-nested-interactive.js

```diff
@@ -19,6 +19,7 @@
 const isInteractiveElement = require('../helpers/is-interactive-element');
 const parseConfig = require('../helpers/parse-interactive-element-config');
 const Rule = require('./base');
+// const { ERROR_MESSAGE } = require('./no-action');
```

Comment:
This is odd, we should either bring it back or delete the commented file.

Reply:
right! I wasn't sure what it was doing, figured we'd go over it in review.